### PR TITLE
MWPW-163170: Non-profits - Adding email localization

### PR DESCRIPTION
**Pull request by joaquinrivero**
- Adding email localization for non profits block

Resolves: [MWPW-163170](https://jira.corp.adobe.com/browse/MWPW-163170)

Test URLs:

Before: https://main--cc--adobecom.hlx.live/nonprofits/apply?martech=off
After: https://mwpw-163170--cc--adobecom.hlx.live/nonprofits/apply?martech=off

### CODE REVIEW

File: creativecloud/blocks/nonprofit/nonprofit.js [hunk 1]
```diff
@@ -159,14 +160,15 @@ async function sendOrganizationData() {
     }
 
     let body;
+    const { eift } = locale;
```
Comment (aishwaryamathuria) [inline creativecloud/blocks/nonprofit/nonprofit.js:163]: Should this be 'ietf'?

File: creativecloud/blocks/nonprofit/nonprofit.js [hunk 2]
```diff
@@ -159,14 +160,16 @@ async function sendOrganizationData() {
     }
 
     let body;
+    const { locale } = getConfig();
+    const ietf = locale.language;
```
Comment (Ruchika4) [inline creativecloud/blocks/nonprofit/nonprofit.js:164]: shouldn't it be locale.ietf instead of locale.language ?

**Review by Ruchika4 — CHANGES_REQUESTED** [review]

File: creativecloud/blocks/nonprofit/nonprofit.js (hunk 2 above)
Comment (joaquinrivero) [reply creativecloud/blocks/nonprofit/nonprofit.js]: Thanks!

**Review by Ruchika4 — APPROVED** [review]

**Review by tedAir — APPROVED** [review]
Verified by filing the form. Somehow didn't receive an email. But I verified the validation-submission call and the language code is sent correctly:  {
  "validationInviteId": "validationinvite_f439c31f-2952-4b4c-8d62-429365695cb3",
  "organisationId": "organisation_000000C701vaJLHWQ6ZOKjd6Rm6G9",
  "firstName": "TestingMX",
  "lastName": "Zeng",
  "email": "ZENG@ADOBE.COM",
  "language": "es-MX"
}
Approved for STAGE.

**Pull request merged**